PostHog/PostHogConfig.swift

```diff
@@ -55,6 +55,16 @@ import Foundation
         @objc public let sessionReplayConfig: PostHogSessionReplayConfig = .init()
     #endif
 
+    #if os(iOS) || targetEnvironment(macCatalyst)
+        /// Enable Autocapture for iOS
+        /// Experimental support
+        /// Default: false
+        @objc public var autocapture: Bool = false
+        /// Autocapture configuration
+        /// Experimental support
+        @objc public let autocaptureConfig: PostHogAutocaptureConfig = .init()
```

Comment:
```suggestion
        @objc public let autoCaptureConfig: PostHogAutocaptureConfig = .init()
```

would that be better?

Reply:
makes sense, i have not found any form of Auto Capture in the docs so lets keep as it is